TombEngine/Renderer/RendererDraw.cpp

```diff
@@ -3141,15 +3164,15 @@ namespace TEN::Renderer
 			if (!levelPtr->GetHorizonEnabled(layer) || levelPtr->GetHorizonTransparency(layer) <= EPSILON)
 				continue;
 
+			_shaders.Bind(Shader::Sky);
```

Comment:
Shader binding was moved upwards. Does it change anything?

Reply:
It seems that it doesn't cause problems